[2021-02-18 22:34:23,942] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-17T00:04:00+00:00 [queued]>
[2021-02-18 22:34:23,948] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-17T00:04:00+00:00 [queued]>
[2021-02-18 22:34:23,948] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:34:23,948] {taskinstance.py:1018} INFO - Starting attempt 1 of 2
[2021-02-18 22:34:23,948] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:34:23,958] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): get_tweets_by_user_list> on 2021-02-17T00:04:00+00:00
[2021-02-18 22:34:23,960] {standard_task_runner.py:51} INFO - Started process 17462 to run task
[2021-02-18 22:34:23,966] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'tweets_by_user', 'get_tweets_by_user_list', '2021-02-17T00:04:00+00:00', '--job-id', '1064', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tweets_by_user.py', '--cfg-path', '/tmp/tmpxe26vml0']
[2021-02-18 22:34:23,968] {standard_task_runner.py:76} INFO - Job 1064: Subtask get_tweets_by_user_list
[2021-02-18 22:34:23,998] {logging_mixin.py:103} INFO - Running <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-17T00:04:00+00:00 [running]> on host ip-172-31-3-59.ec2.internal
[2021-02-18 22:34:24,020] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tweets_by_user
AIRFLOW_CTX_TASK_ID=get_tweets_by_user_list
AIRFLOW_CTX_EXECUTION_DATE=2021-02-17T00:04:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-17T00:04:00+00:00
[2021-02-18 22:34:24,021] {logging_mixin.py:103} INFO - Autenticação no twitter
[2021-02-18 22:34:24,053] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:34:24,212] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:34:24,630] {logging_mixin.py:103} INFO - 200
[2021-02-18 22:34:24,638] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:34:24,776] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:34:25,198] {logging_mixin.py:103} INFO - 400
[2021-02-18 22:34:25,207] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:34:25,379] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:34:25,983] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:34:25,987] {logging_mixin.py:103} INFO - 0:00:01.965994
[2021-02-18 22:34:25,987] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:34:27,922] {logging_mixin.py:103} INFO - 0:00:01.935698
[2021-02-18 22:34:27,938] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:34:27,959] {logging_mixin.py:103} INFO - 0:00:00.036626
[2021-02-18 22:34:27,959] {logging_mixin.py:103} INFO - 600 devidamente inseridas no banco
[2021-02-18 22:34:27,961] {python.py:118} INFO - Done. Returned value was: None
[2021-02-18 22:34:27,964] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=tweets_by_user, task_id=get_tweets_by_user_list, execution_date=20210217T000400, start_date=20210218T223423, end_date=20210218T223427
[2021-02-18 22:34:27,984] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-18 22:34:27,990] {local_task_job.py:118} INFO - Task exited with return code 0
